src/test/java/org/xstefank/whitelist/RemoveUserCommandTest.java

```diff
@@ -0,0 +1,37 @@
+package org.xstefank.whitelist;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.xstefank.utils.TestUtils;
+
+public class RemoveUserCommandTest {
+
+    private RemoveUserCommand removeUserCommand;
+
+    @Before
+    public void before() {
+        TestUtils.USER_LIST.addUser("prUser");
```

Comment:
not a good practice to modify public static final property in different class just for the test purposes